### ISSUE TITLE
Missing: global app menu

### ISSUE THREAD

**Issue by X-Ryl669** [mpsq/arewewaylandyet#148]
Right now, there's no common way to have a global application menu (instead of per application menu bar), like we have in X11.
So any environment like Unity or Mac OSX aren't possible in Wayland.

For KDE application, it seems to work, but not for Gnome based application like Firefox.

**Comment by fenglengshun** [comment]
Yeah, this is something I miss dearly with many apps when run on Wayland. Global Menu just save spaces for me. It also provides some additional functionalities, like with Chromium browsers which exposes some things for quicker access at my top panel. Also, some things are outright broken because of lack of Global Menu support - I have to run OBS in xwayland every time I want to access remux menu and everything else not accessible directly from the app.

<details><summary>Details</summary>

![image](https://github.com/mpsq/arewewaylandyet/assets/26621899/b6e6ec06-7686-45f1-a1e4-f7b913870610)

![image](https://github.com/mpsq/arewewaylandyet/assets/26621899/9a772b5a-73ff-4565-a84a-3cf88b1b3f40)

</details>
